includes/core/API/class-wc_pay_request.php

```diff
@@ -0,0 +1,263 @@
+<?php
+namespace WCPay\Core\API;
+
+use WC_Payments;
+use WC_Payments_Http;
+use WC_Payments_Utils;
+use WCPay\Core\Contracts\API\Request\BaseRequest;
+use WCPay\Core\DataTransferObjects\API\Request\Create_Charge;;
+use WCPay\Core\DataTransferObjects\API\Request\Request;
+use WCPay\Core\DataTransferObjects\Response;
+use WCPay\Core\Enums\Http_Methods;
+use WCPay\Core\Enums\Wc_Pay_Endpoints;
+use WCPay\Core\Factories\Create_Charge_Dto;
+use WCPay\Exceptions\Amount_Too_Small_Exception;
+use WCPay\Exceptions\API_Exception;
+use WCPay\Fraud_Prevention\Fraud_Prevention_Service;
+use WCPay\Logger;
+
+class Wc_Pay_Request
```

Comment:
How is this used?

Reply:
I removed this code for a sake of simplicity and used original wc pay http api class. We could add our own implementation if needed.